packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -1,23 +1,39 @@
 import { css, type SerializedStyles } from '@emotion/react';
 import { customProperties } from '@commercetools-uikit/design-system';
+import type { TModalPageSize } from './modal-page';
 
 export const TRANSITION_DURATION = 200;
 
 type StyleProps = {
   zIndex?: number;
-  size?: 'small' | 'large';
+  size?: TModalPageSize;
+};
+
+const getContainerSize = (size?: TModalPageSize) => {
+  switch (size) {
+    case 10:
+      return '600px !important';
+    case 20:
+      return '900px !important';
+    case 30:
+      return '1200px !important';
+    case 'scale':
+      return '100%';
+    default:
+      return '100%';
+  }
 };
 
 export const getContainerStyles = (_props: StyleProps): SerializedStyles => css`
   position: absolute;
   top: 0;
   right: 0;
   height: 100%;
-  width: ${_props.size === 'small' ? '600px !important' : '100%'};
+  width: ${getContainerSize(_props.size)};
   display: flex;
   flex-direction: column;
   background-color: ${customProperties.colorSurface};
-  box-shadow: ${customProperties.shadow4}, ${customProperties.shadow6};
+  box-shadow: 0px 0px 40px 0px rgba(0, 0, 0, 0.1);
```

Comment:
@CarlosCortizasCT  This is the box-shadow value that Filip wanted. I added it here.